api/contract/cli.yml

```diff
@@ -253,3 +255,11 @@ components:
       $ref: "./openapi/src/common/schemas/Cells.yml"
     Cell:
       $ref: "./openapi/src/common/schemas/Cell.yml"
+    TemplateExport:
+      $ref: "./overrides/schemas/TemplateExport.yml"
+    Template:
+      $ref: "./openapi/src/common/schemas/Template.yml"
```

Comment:
Is it a problem that these `openapi/src/` paths aren't present in `api/contract`?

Reply:
Hmmm where did you see the paths weren't present? `openapi` is a git submodule so it takes a `git submodule update --init` for the paths to appear locally. The UI renders the folder as a link to the other repo.